src/SwedbankPay.Sdk/PaymentInstruments/Card/CardPaymentAuthorizationRequest.cs

```diff
@@ -1,17 +1,17 @@
-﻿namespace SwedbankPay.Sdk.Payments.CardPayments
-{
-    public class CardPaymentAuthorizationRequest
-    {
-        public CardPaymentAuthorizationRequest(string cardNumber,
-                                    int cardExpiryMonth,
-                                    int cardExpiryYear,
-                                    string cardVerificationCode = null,
-                                    string cardHolderName = null)
-        {
-            Transaction = new CardPaymentAuthorizationRequestTransaction(cardNumber, cardExpiryMonth, cardExpiryYear, cardVerificationCode, cardHolderName);
-        }
-
-
-        public CardPaymentAuthorizationRequestTransaction Transaction { get; }
-    }
+﻿namespace SwedbankPay.Sdk.Payments.CardPayments
+{
+    public class CardPaymentAuthorizationRequest
+    {
+        public CardPaymentAuthorizationRequest(string cardNumber,
+                                    int cardExpiryMonth,
+                                    int cardExpiryYear,
+                                    string cardVerificationCode = null,

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)